### DISCUSSION TITLE
How to bring back offline channel?

### DISCUSSION THREAD

**Discussion by Darth-Coin** [ACINQ/eclair#2498] [Q&A] [answered]
Hi, I was doing some [tests with many different LN wallet apps](https://github.com/hsjoberg/blixt-wallet/issues/1065) and one is Phoenix.
Sending many different amounts from other wallets to Phoenix I end up with 3 channels, like in bellow image, and one is always offline.
I am using latest version 1.4.23 (38) on Android 8.1 device.
I tried restarting, I tried clicking on the offline channel details but is not showing anything.
All other channels are functional, I can make payments with them.
Is there a procedure to bring back online that channel?

![image](https://user-images.githubusercontent.com/56997124/203016437-d03742e2-8b5a-4c1b-871c-c64c30ae5285.png)

Side note: you have a scammer impersonating acinq support on telegram. Report it, but just FYI is sending people to a scam webpage.

**Comment by t-bast** [comment, marked as answer]
This is not the right repository for phoenix-related issues, eclair is our server node implementation and is independent of Phoenix.
Please open an issue on the Phoenix repo: https://github.com/acinq/phoenix

**Reply by pm47** [reply]
Feel free to also open a ticket on phoenix@acinq.co with your node id if you don't want to post it publicly.

Looks like this channel was superseded by the 3rd one with the exact same capacity. Phoenix has not realized that this channel doesn't exist.

**Reply by Darth-Coin** [reply]
Thanks, I didn't open a issue, because I am not sure if is a bug, maybe is just a glitch that I can fix it myself. 
So I posted first as a discussion
https://github.com/ACINQ/phoenix/discussions/315
